build.assets/Makefile

```diff
@@ -36,6 +36,7 @@ NODE_VERSION ?= 16.13.2
 RUST_VERSION ?= 1.63.0 # don't bump this without checking GLIBC compatibility
 BORINGCRYPTO_RUNTIME=$(GOLANG_VERSION)b7
 LIBBPF_VERSION ?= 0.7.0-teleport
+LIBPCSCLITE_VERSION ?= 1.9.8
```

Comment:
Can you rename the tag to `1.9.8-teleport` to indicate that we're not using the mainstream version? The same as we did with `libbpf` line above.